lib/providers/transit.js

```diff
@@ -79,6 +79,19 @@ class TransitProvider {
         return response;
       });
   }
+
+  /**
+   * Get a unique ID for the given token and secret
+   *
+   * @param {String} token
+   * @param {Object} secret
+   * @param {Object} secret.key
+   * @param {Object} secret.ciphertext
+   * @return {String}
+   */
+  static getSecretID(token, secret) {
```

Comment:
I removed the handling of `Object` secrets and related tests in https://github.com/rapid7/tokend/pull/62/commits/ceaba809abfefc2a757e11369aacbac587aa86c4.